portal-web/test/functional/com/liferay/portalweb/macros/WorkflowMetrics.macro

```diff
@@ -81,4 +107,194 @@ definition {
+				value1 = "inst / month");
+
+			WorkflowMetrics.AssertChartIsPresent(chartVelocityUnit = "Inst / Month");
+
+			Click(locator1 = "WorkflowMetrics#METRICS_COMPLETION_VELOCITY_UNIT_DROPDOWN");
+
+			AssertElementNotPresent(locator1 = "WorkflowMetrics#METRICS_COMPLETION_VELOCITY_UNIT_DROPDOWN_OPTION_INST_HOUR");
+
+			AssertElementNotPresent(locator1 = "WorkflowMetrics#METRICS_COMPLETION_VELOCITY_UNIT_DROPDOWN_OPTION_INST_YEAR");
+		}
+
+		if ("${key_completionVelocityDateRange}" == "Last 180 Days") {
+			AssertElementPresent(locator1 = "WorkflowMetrics#METRICS_COMPLETION_VELOCITY_UNIT_DROPDOWN_OPTION_INST_MONTH");
+
+			AssertTextEquals(
+				locator1 = "WorkflowMetrics#METRICS_COMPLETION_VELOCITY_CHART_UNIT_LABEL",
+				value1 = "inst / month");
+
+			WorkflowMetrics.AssertChartIsPresent(chartVelocityUnit = "Inst / Month");
+
+			Click(locator1 = "WorkflowMetrics#METRICS_COMPLETION_VELOCITY_UNIT_DROPDOWN");
+
+			Click(locator1 = "WorkflowMetrics#METRICS_COMPLETION_VELOCITY_UNIT_DROPDOWN_OPTION_INST_WEEK");
+
+			AssertTextEquals(
+				locator1 = "WorkflowMetrics#METRICS_COMPLETION_VELOCITY_CHART_UNIT_LABEL",
+				value1 = "inst / week");
+
+			WorkflowMetrics.AssertChartIsPresent(chartVelocityUnit = "Inst / Week");
+
+			Click(locator1 = "WorkflowMetrics#METRICS_COMPLETION_VELOCITY_UNIT_DROPDOWN");
+
+			AssertElementNotPresent(locator1 = "WorkflowMetrics#METRICS_COMPLETION_VELOCITY_UNIT_DROPDOWN_OPTION_INST_HOUR");
+
+			AssertElementNotPresent(locator1 = "WorkflowMetrics#METRICS_COMPLETION_VELOCITY_UNIT_DROPDOWN_OPTION_INST_DAY");
+
+			AssertElementNotPresent(locator1 = "WorkflowMetrics#METRICS_COMPLETION_VELOCITY_UNIT_DROPDOWN_OPTION_INST_YEAR");
+		}
+
+		if ("${key_completionVelocityDateRange}" == "Last Year") {
+			AssertElementPresent(locator1 = "WorkflowMetrics#METRICS_COMPLETION_VELOCITY_UNIT_DROPDOWN_OPTION_INST_WEEK");
+
+			AssertTextEquals(
+				locator1 = "WorkflowMetrics#METRICS_COMPLETION_VELOCITY_CHART_UNIT_LABEL",
+				value1 = "inst / week");
+
+			WorkflowMetrics.AssertChartIsPresent(chartVelocityUnit = "Inst / Week");
+
+			Click(locator1 = "WorkflowMetrics#METRICS_COMPLETION_VELOCITY_UNIT_DROPDOWN");
+
+			Click(locator1 = "WorkflowMetrics#METRICS_COMPLETION_VELOCITY_UNIT_DROPDOWN_OPTION_INST_MONTH");
+
+			AssertTextEquals(
+				locator1 = "WorkflowMetrics#METRICS_COMPLETION_VELOCITY_CHART_UNIT_LABEL",
+				value1 = "inst / month");
+
+			WorkflowMetrics.AssertChartIsPresent(chartVelocityUnit = "Inst / Month");
+
+			Click(locator1 = "WorkflowMetrics#METRICS_COMPLETION_VELOCITY_UNIT_DROPDOWN");
+
+			AssertElementNotPresent(locator1 = "WorkflowMetrics#METRICS_COMPLETION_VELOCITY_UNIT_DROPDOWN_OPTION_INST_HOUR");
+
+			AssertElementNotPresent(locator1 = "WorkflowMetrics#METRICS_COMPLETION_VELOCITY_UNIT_DROPDOWN_OPTION_INST_DAY");
+
+			AssertElementNotPresent(locator1 = "WorkflowMetrics#METRICS_COMPLETION_VELOCITY_UNIT_DROPDOWN_OPTION_INST_YEAR");
+
+		}
+
+	}
```

Comment:
hi @barrosdennys , in general, I would suggest you check if we could reuse some similar calls instead of repeating them.